security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,37 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-4-0"]
+== OpenShift Compliance Operator 1.4.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.4.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:7658[RHBA-2023:7658 - OpenShift Compliance Operator bug fix and enhancement update]
+
+[id="compliance-operator-1-4-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* With this update, clusters which use custom node pools outside the default `worker` and `master` node pools no longer need to supply additional variables to ensure Compliance Operator aggregates the configuration file for that node pool.
+
+* Users can now pause scan schedules by setting the `ScanSetting.suspend` attribute to `True`. This allows users to suspend a scan schedule and reactivate it without the need to delete and re-create the `ScanSettingBinding`. This simplifies pausing scan schedules during maintenance periods. (link:https://issues.redhat.com/browse/CMP-2123[*CMP-2123*])
+
+* Compliance Operator now supports an optional `version` attribute on `Profile` custom resources. (link:https://issues.redhat.com/browse/CMP-2125[*CMP-2125*])
+
+* Compliance Operator now supports profile names in `ComplianceRules`. (link:https://issues.redhat.com/browse/CMP-2126[*CMP-2126*])
+
+* Compliance Operator compatibility with improved `cronjob` API improvements is available in this release. (link:https://issues.redhat.com/browse/CMP-2310[*CMP-2310*])
+
+[id="compliance-operator-1-4-0-bug-fixes"]
+=== Bug fixes
+
+* With this update, `rprivate` default mount propagation is now handled correctly for root volume mounts of pods that rely on multipathing. (link:https://issues.redhat.com/browse/OCPBUGS-17494[*OCPBUGS-17494*])
+
+* Previously, Compliance Operator remediations would show as `unapplied` after scans for compliance completed. With this release, remediations are now correctly shown after compliance scans complete. (link:https://issues.redhat.com/browse/OCPBUGS-8041[*OCPBUGS-8041*])
+
+* Before this update, rule `rhcos4-audit-rules-login-events-faillock` would fail even after auto-remediation has been applied. With this update, `rhcos4-audit-rules-login-events-faillock` failure locks are now applied correctly after auto-remediation. (link:https://issues.redhat.com/browse/OCPBUGS-24594[*OCPBUGS-24594*])
+
+* Previously, upgrades from Compliance Operator 1.3.1 to Compliance Operator 1.4.0 would cause OVS rules scan results to go from `PASS` to `NOT-APPLICABLE`. With this update, OVS rules scan results now show `PASS` (link:https://issues.redhat.com/browse/OCPBUGS-25323[*OCPBUGS-25323*])
```

Comment:
This one is tricky since we caught it before the release, and before users could upgrade from 1.3.1 to 1.4.0. Users shouldn't seen any issues with OVS rules, on 1.4.0 compared to 1.3.1.

Reply:
@rhmdnd Oh dang, OK. Leave it as is, or delete?